packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```
@@ -41,11 +54,19 @@ BuildRequires: %{_cross_os}kernel-6.1-archive
 %description
 %{summary}.
 
+%package -n fabricmanager
```

We don't want a package that's just called "fabricmanager":
```suggestion
%package fabric-manager
```

---

changed as suggested